src/sdk/pynni/nni/compression/torch/utils/counter.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import torch
+import torch.nn as nn
+from torch.nn.modules.conv import _ConvNd
+from torch.nn.modules.batchnorm import _BatchNorm
+from nni.compression.torch.compressor import PrunerModuleWrapper
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from thop import profile
+except ImportError:
+    _logger.warning('Please install thop using command: pip install thop')
+    
+def count_flops_params(model: nn.Module, input_size=None, verbose=True):
```

Comment:
suggest to remove default value of input_size

Reply:
fix it. please review it on the latest version~